### PR TITLE
add files for network lb rule migration

### PR DESCRIPTION
**Pull request by philshin-google**

### CODE REVIEW

File: dashboards/google-network-lb-rule/network-lb-rule.json [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+            "monitoredResources": [
+              {
+                "type": "tcp_lb_rule",
+                "labels": {}
+              }
+            ],
+            "policyNames": []
+          },
+          "title": "Incidents",
+          "id": ""
```
Comment (EvanSimpson) [inline dashboards/google-network-lb-rule/network-lb-rule.json:20]:
```suggestion
          "title": "Incidents"
```

File: dashboards/google-network-lb-rule/network-lb-rule.json [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [],
+                      "perSeriesAligner": "ALIGN_RATE"
+                    },
+                    "filter": "metric.type=\"loadbalancing.googleapis.com/l3/external/egress_packets_count\" resource.type=\"udp_lb_rule\" resource.label.\"load_balancer_name\"=\"tcp-load-balancer\" resource.label.\"project_id\"=\"gstackdriver-e2e-turtle\" resource.label.\"region\"=\"us-east1\""
```
Comment (EvanSimpson) [inline dashboards/google-network-lb-rule/network-lb-rule.json:249]: Remove resource labels here
```suggestion
                    "filter": "metric.type=\"loadbalancing.googleapis.com/l3/external/egress_packets_count\" resource.type=\"udp_lb_rule\""
```

File: dashboards/google-network-lb-rule/network-lb-rule.json [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+              {
+                "minAlignmentPeriod": "60s",
+                "plotType": "HEATMAP",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [],
```
Comment (EvanSimpson) [inline dashboards/google-network-lb-rule/network-lb-rule.json:283]: No group bys? Intentional?

File: dashboards/google-network-lb-rule/network-lb-rule.json [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+          }
+        },
+        "height": 16,
+        "width": 24,
+        "yPos": 44
+      }
+    ]
+  },
+  "dashboardFilters": [],
+  "labels": {}
```
Comment (EvanSimpson) [inline dashboards/google-network-lb-rule/network-lb-rule.json:452]:
```suggestion
  }
```

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/google-network-lb-rule/network-lb-rule.json (hunk 3 above)
Comment (philshin-google) [reply dashboards/google-network-lb-rule/network-lb-rule.json:282]: group bys are not possible with the distribution aggregation for heat maps. Given the metric is latency, for a heat map chart, do you think no group bys make sense?

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**